Source/ARTPushRecipient.h

```diff
@@ -0,0 +1,47 @@
+//
+//  ARTPushRecipient.h
```

Comment:
Sorry you took all the effort to do this, but we really want to keep this "stringly-typed" so that adding new recipient types on the server don't require changing anything in the client libraries.

So ARTPushRecipient should just be NSDictionary.